package.json

```diff
@@ -52,7 +52,9 @@
     "start": "node docs/server.js port=3000",
     "start_docs": "node docs/server.js port=25907",
     "test": "gulp test",
-    "watch": "gulp watch"
+    "watch": "gulp watch",
+    "preprotractor": "webdriver-manager update",
+    "protractor": "protractor benchmarks/protractor.conf.js --specs benchmarks/dist/benchmark.spec.js"
```

Comment:
I'm not sure it's a good idea move everything inside `npm script` also because `protractor benchmarks/protractor.conf.js --specs benchmarks/dist/benchmark.spec.js` it's a possible temporary solution because benchpress it's still in working progress.
I'm sorry I'm not in general a big fun put everything inside `npm` script.